wp-content/plugins/core/src/Theme/Config/Web_Fonts.php

```diff
@@ -2,15 +2,18 @@
 
 namespace Tribe\Project\Theme\Config;
 
+/**
+ * @TODO rename to a proper class
+ */
 class Web_Fonts {
 
-	public const  PROVIDER_TYPEKIT = 'typekit';
-	public const  PROVIDER_GOOGLE  = 'google';
 	public const  PROVIDER_CUSTOM  = 'custom';
+	public const  PROVIDER_GOOGLE  = 'google';
+	public const  PROVIDER_TYPEKIT = 'typekit';
 	private const TYPEKIT_API      = 'https://use.typekit.net';
 
 	/**
-	 * @var array
+	 * @var array<string, string>
```

Comment:
Same as previous comment. This is different than other comments I've seen.

Reply:
According to [the existing comments I read](https://github.com/moderntribe/square-one/blob/9a65e37cc838f6a1d7c0a6180d98c0a0e2338644/wp-content/plugins/core/src/Theme/Theme_Definer.php#L119) these will be an array with a string as the key and a string as the value, so this is accurate.

These are PHPStan doc types: https://phpstan.org/writing-php-code/phpdoc-types